### ISSUE TITLE
Failure of wf 140.54 after integration of #27887

### ISSUE THREAD

**Issue by fabiocos** [cms-sw/cmssw#27919]
The test wf 140.54 is reproducibly failing since CMSSW_11_0_X_2019-08-30-2300 after the integration of #27887. This PR modifies the jet reconstruction sequences, adopting tasks, but this does not appear to properly work for the pA_2016 modifier involved in the 140.54 test.

The error is:
```
----- Begin Fatal Exception 31-Aug-2019 01:03:17 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 211313 lumi: 1 event: 17754 stream: 2
   [1] Calling method for module BTagPerformanceAnalyzerOnData/'bTagAnalysis'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: reco::JetCorrector
Looking for module label: ak4PFCHSL1FastL2L3ResidualCorrector
Looking for productInstanceName: 

----- End Fatal Exception -------------------------------------------------
```

where the DQM module https://cmssdt.cern.ch/lxr/source/DQMOffline/RecoB/python/bTagAnalysisData_cfi.py 
fails to get as input a product of the module ak4PFCHSL1FastL2L3ResidualCorrector. 
This is related to the recent update of ```JetCorrectorsFromReco```

**Comment by fabiocos** [comment]
assign reconstruction

@jeongeun FYI

**Comment by perrotta** [comment]
@jeongeun could you please have a look?

**Comment by jeongeun** [comment]
Sorry for the late response. 
I found 1 file  `cmssw/Configuration/StandardSequences/python/Reconstruction_Data_cff.py` which uses `jetCorrectorsForReco`.
I think the *last line* in Reconstruction_Data_cff.py file might create the errors in 140.54.  
 `jetCorrectorsForReco.replace(ak4PFCHSL1FastL2L3CorrectorChain, ak4PFCHSL1FastL2L3ResidualCorrectorChain)`)
Fix => **Change Chain to Task**
Now I'm doing runtest and if the outputs look okay, then I will fix this `Reconstruction_Data_cff.py`.

**Comment by fabiocos** [comment]
the problem is solved by #27931
